sources/packages/api/src/route-controllers/user/user.controller.ts

```diff
@@ -120,4 +121,16 @@ export class UserController extends BaseController {
       throw error;
     }
   }
+
+  @Get("/institutionUser")
+  async institutionDetail(
+    @UserToken() userToken: IUserToken,
+  ): Promise<InstitutionUserDto> {
+    const user = await this.service.getActiveUser(userToken.userName);
```

Comment:
Please add a check if the user is present, if not, throw an HTTP error exception.

Reply:
User is validated and exception is thrown accordingly